cinema/tests/test_movie_api.py

```diff
@@ -157,3 +158,179 @@ def test_image_url_is_shown_on_movie_session_detail(self):
         res = self.client.get(MOVIE_SESSION_URL)
 
         self.assertIn("movie_image", res.data[0].keys())
```

Comment:
Code Quality Issue: This test case could be improved by also verifying that the correct image URL is returned in the response.